src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
-::The Gutter - Intro <1156,67>
-Anna was right. That piece of shit was trying to kill me one way or another. I guess I’d known it all along.
+<!-- If using debug menu shortcuts, prompts player to enter their name -->
+
+::The Gutter - Name prompt
+{ (set: $playerName to (prompt: "Enter name:", ""))
+(if: (text: $playerName)'s length > 0)[Right, my name is $playerName. [[Moving on|The Gutter - Outside approach]].{
+  (if: $playerName's last is "s")[
+    (set: $playerNamePlural to ($playerName + "es"))
+  ]
+  (else-if: ($playerName's last is "h") and ($playerName's 2ndlast is "s"))[
+    (set: $playerNamePlural to ($playerName + "es"))
+  ]
+  (else-if: ($playerName's last is "h") and ($playerName's 2ndlast is "c"))[
+    (set: $playerNamePlural to ($playerName + "es"))
+  ]
+  (else:)[
+    (set: $playerNamePlural to ($playerName + "s"))
+  ]
+}]
+}(else:)[No, [[that's not right|The Gutter - Name prompt]].]
+
+(set: $dissolve to (transition: 'dissolve') + (transition-time: 3s))
+(set: $hatchet to true)(set: $multitool to true)
```

Comment:
Probably outside the scope of this game, but in the future, we should look into adding a debug menu that lets you toggle these variables on/off at any point. It wouldn't be that hard — just add a link to a debug passage within the header or footer that includes toggles for each of these. For now, though, just having access to both these tools is totally good.

Reply:
Yeah, I could see anything larger than the scope of what we've built benefiting a lot from that type of debug menu. We would probably benefit for this project too, given time, but I agree it's outside of scope at this point. Hadn't thought of making toggles, that's a cool idea!